packages/components/icons/src/v5/ArrowCounterClockwise.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ArrowCounterClockwise as PhosphorIcon } from '@phosphor-icons/react';
+
+import {
+  IconVariant,
+  generateForma36Icon,
+  wrapPhosphorIcon,
+} from '../../../icon/src/generateIcon';
+
+const ArrowCounterClockwise = wrapPhosphorIcon(PhosphorIcon);
+
+export const ArrowCounterClockwiseIcon = generateForma36Icon({
+  variants: {
+    [IconVariant.Active]: (props) => (
+      <ArrowCounterClockwise {...props} weight="fill" />
+    ),
+    [IconVariant.Default]: ArrowCounterClockwise,
+  },
```

Comment:
It would be great if we could abstract the variant logic within the generator to make our contributors' lives easier and less prone to errors.

Following the [design](https://www.figma.com/file/ILoICAP1fxyvE7xmpWiJBi/Forma-36-Assets?type=design&node-id=1699-22&mode=design&t=1LzNdfhx1togJCsI-0), we would always have the same set of icons (3 sizes, 2 variants (default/active)).

If it turns out to be too tricky, we could create a script to generate the icon automatically given the Phosphor name, wdyt?

Reply:
Yeah I hear you and will give it some thought. To be able to solve multiple use cases where we want to render different SVGs based on a component prop, I'm thinking we might need to expose a function like this, where variants are set explicitly, and another level of abstraction that achieves what you are suggesting.

Reply:
I added another level of abstraction.

Reply:
Thank you, this is much cleaner! ✨ 

I can see the addition of a script to automatically generate the icon files/changes based on the icon(s) contributors want to add. 

I will review it thoroughly when I am back, but feel free to go ahead if other folks review & approve 👍